src/Nethermind/Nethermind.Network/ProtocolsManager.cs

```diff
@@ -412,6 +412,15 @@ public void AddSupportedCapability(Capability capability)
 
             _capabilities.Add(capability);
         }
+        
+        public void RemoveSupportedCapability(Capability capability)
+        {
+            if (_capabilities.Contains(capability))
```

Comment:
Maybe _capabilities should be HashSet?
Will that break order? Is order of capabilities needed?